### PR TITLE
Close #28469: Update notification permission prompt onboarding strings

### LINKED ISSUE #28469: Add notification permission prompt onboarding strings
This is to add the strings needed for https://bugzilla.mozilla.org/show_bug.cgi?id=1808874

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-22254)

### PR DESCRIPTION
**Pull request by rocketsroger**
Update for the new design coming from Product.
It will now have only one button that says continue.

prompt_notification

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### QA

- [ ] **QA Needed**

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-debug` task.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation
Fixes #28469

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -279,10 +279,8 @@
     <string name="onboarding_home_enable_notifications_title" tools:ignore="UnusedResources">Notifications help you do more with Firefox</string>
     <!-- Enable notification pre permission dialog description with rationale -->
     <string name="onboarding_home_enable_notifications_description" tools:ignore="UnusedResources">Sync your tabs between devices, manage downloads, get tips about making the most of Firefox’s privacy protection, and more.</string>
-    <!-- Text for the button to enable app notifications on the device -->
-    <string name="onboarding_home_enable_notifications_positive_button" tools:ignore="UnusedResources">Enable notifications</string>
-    <!-- Text for the button to not enable app notifications on the device and dismiss the dialog -->
-    <string name="onboarding_home_enable_notifications_negative_button" tools:ignore="UnusedResources">No thanks</string>
```
Comment (MozillaNoah) [inline app/src/main/res/values/strings.xml:285]: these need the `moz:removedIn="XXX"` treatment.

Comment (rocketsroger) [reply app/src/main/res/values/strings.xml:285]: That's a good point.  However, there's a change in the requirement again.  I'll be putting these two strings back.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -279,10 +279,8 @@
...
+    <!-- Text for the button to request for notification permission on the device -->
+    <string name="onboarding_home_enable_notifications_continue_button" tools:ignore="UnusedResources">Continue</string>
```
[the remark this answers is not in the document]
Comment (rocketsroger) [reply app/src/main/res/values/strings.xml:283]: This was landed yesterday in preparation for a feature.  I'll update.  Thanks

**Comment by rocketsroger** [comment]
Updated to match @Kate-Galetski 's latest design.  With Jeff Pfaller's confirmation on the strings.

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (MozillaNoah) [reply app/src/main/res/values/strings.xml:285]: what's the sitch on these two strings? Are they out?

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -279,10 +279,10 @@
     <string name="onboarding_home_enable_notifications_title" tools:ignore="UnusedResources">Notifications help you do more with Firefox</string>
     <!-- Enable notification pre permission dialog description with rationale -->
     <string name="onboarding_home_enable_notifications_description" tools:ignore="UnusedResources">Sync your tabs between devices, manage downloads, get tips about making the most of Firefox’s privacy protection, and more.</string>
-    <!-- Text for the button to enable app notifications on the device -->
-    <string name="onboarding_home_enable_notifications_positive_button" tools:ignore="UnusedResources">Enable notifications</string>
-    <!-- Text for the button to not enable app notifications on the device and dismiss the dialog -->
-    <string name="onboarding_home_enable_notifications_negative_button" tools:ignore="UnusedResources">No thanks</string>
+    <!-- Text for the button to request notification permission on the device -->
+    <string name="onboarding_home_enable_notifications_positive_button" tools:ignore="UnusedResources">Continue</string>
```
Comment (MozillaNoah) [inline app/src/main/res/values/strings.xml:283]:
```suggestion
    <string name="onboarding_home_enable_notifications_positive_button">Continue</string>
```

Comment (rocketsroger) [reply app/src/main/res/values/strings.xml:283]: I do still need the "UnusedResources" to pass lint.  The feature is coming but nothing is using this button right now.

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (rocketsroger) [reply app/src/main/res/values/strings.xml:285]: I landed these strings just yesterday.  I'm not sure if I need to add the _2 if they have not been translated and used do I?

File: app/src/main/res/values/strings.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (rocketsroger) [reply app/src/main/res/values/strings.xml]: Confirmed with @Delphine that this is good to land without "_2"  thanks

**Review by MozillaNoah — APPROVED** [review]

**Pull request merged**